testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -781,12 +777,14 @@
 
 # Image-specific steps
 When(/^I enter "([^"]*)" relative to profiles as "([^"]*)"$/) do |path, field|
-  step %(I enter "#{$git_profiles}/#{path}" as "#{field}")
+  git_profiles = ENV['GITPROFILES']
```

Comment:
I think we should move that variable outside the method, we can have it with the rest of global variables which retrieves values from ENV

Reply:
the purpose is precisely to undo that!

Reply:
I don't see how that global variable can hurt :man_shrugging:  
I only can see the benefit of not read it from the system every time. I see the purpose of reducing global variables if those can change its value, but when the value will never change, I see benefits.
Anyway, fine to let it as it is.

Reply:
We already have a global variable: `ENV`. It's as fast and as easy to use it as to copy it elsewhere with no point.

We don't have a `$portus_profile` either and it never shocked you...

Reply:
Ah! ENV is a global variable? Somehow in my mind I though it was accessing to the system to retrieve the variable using ENV. Then we are fine, we just read from our stack 😅
Btw, I'm not familiar with that portus profile thing, someday you will need to explain me what's that :)